src/rez/rex.py

```diff
@@ -684,6 +686,59 @@ def get_key_token(self, key):
         # here because the API requires it.
         return "${%s}" % key
 
+    def adjust_env_for_platform(self, env):
+        """ Make required platform-specific adjustments to env.
+        """
+        if sys.platform.startswith('win'):
+            env = self._add_systemroot_to_env_win32(env)
+        return env.copy()
+
+    def _add_systemroot_to_env_win32(self, env):
+        """ Sets ``%SYSTEMROOT%`` environment variable, if not present.
+
+        Args:
+            env (dict): desired environment variables
+
+        Notes:
+            on windows, python-3.6 startup fails within an environment
+            where it ``%PATH%`` includes python3, but ``%SYSTEMROOT%`` is not
+            present.
+
+            for example.
+
+            .. code-block:: python
+
+                from subprocess import Popen
+                cmds = ['python', '--version']
+
+                # successful
+                Popen(cmds)
+                Popen(cmds, env={'PATH': 'C:\\Python-3.6.5',
+                                 'SYSTEMROOT': 'C:\Windows'})
+
+                # failure
+                Popen(cmds, env={'PATH': 'C:\\Python-3.6.5'})
+
+                #> Fatal Python Error: failed to get random numbers to initialize Python
+
+        """
+        # 'SYSTEMROOT' only relevant on windows
+        if not sys.platform.startswith('win'):
```

Comment:
this check is already done, and this is a non-public func, so this can be removed

Reply:
fixed!

thank you, I'm never sure how much I should worry about accidents. this seems like a good rule of thumb.